data/BSatishSutar.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Satish Sutar",
+  "type": "personal",
+  "bio": "Non-IT turned **Digital Nomad** Travel enthusiast Yoga & Tech aficionado"
+  "links": [
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/satish-sutar/",
+      "icon": "FaLinkedIn"
```

Comment:
```suggestion
      "icon": "FaLinkedin"
```

Reply:
Chinmay,

I am unable to find that JSON file to make corrections.
Please advise!
I have checked in branch I have created in my repo and Eddie's repo too...
I am learning GitHub so kindly guide me.
Thanks Chinmay,

Reply:
Hey Chinmay, I found the file finally and I have made corrections as suggested. Thank you!